.github/workflows/build-hw-reusable.yml

```diff
@@ -40,19 +40,21 @@ env:
 
 jobs:
   adapter-build-hw:
-    name: Build & Test HW
+    name: Build & CTS
     if: github.repository == 'oneapi-src/unified-runtime'  # run only on upstream; forks won't have the HW
     strategy:
       matrix:
-        adapter: [{
-          name: "${{inputs.adapter_name}}",
-          other_name: "${{inputs.other_adapter_name}}",
-          platform: "${{inputs.platform}}",
-          static_Loader: "${{inputs.static_loader}}",
-          static_adapter: "${{inputs.static_loader}}"
-        }]
-        build_type: [Debug, Release]
-        compiler: [{c: gcc, cxx: g++}, {c: clang, cxx: clang++}]
+        adapter: [
+          {
+            name: "${{inputs.adapter_name}}",
+            other_name: "${{inputs.other_adapter_name}}",
+            platform: "${{inputs.platform}}",
+            static_Loader: "${{inputs.static_loader}}",
+            static_adapter: "${{inputs.static_loader}}"
+          }
+        ]
+        build_type: [Release]
+        compiler: [{c: gcc, cxx: g++}]
         # TODO: The latest L0 loader segfaults when built with clang.
```

Comment:
the exclusions below with clang should be removed now

Reply:
hmm maybe even all exclusions should be removed? something here is causing L0 static job failing, I think

Reply:
I've removed those, we'll see how that affects the L0 static job.